pkg/metrics/mapmetrics/mapmetrics.go

```diff
@@ -10,33 +10,22 @@ import (
 )
 
 var (
-	MapDrops = prometheus.NewCounterVec(prometheus.CounterOpts{
-		Namespace:   consts.MetricsNamespace,
-		Name:        "map_drops_total",
-		Help:        "The total number of entries dropped per LRU map.",
-		ConstLabels: nil,
-	}, []string{"map"})
 	MapSize = metrics.NewBPFGauge(prometheus.NewDesc(
-		prometheus.BuildFQName(consts.MetricsNamespace, "", "map_in_use_gauge"),
+		prometheus.BuildFQName(consts.MetricsNamespace, "", "map_in_use"),
 		"The total number of in-use entries per map.",
 		[]string{"map", "total"}, nil,
 	))
 	MapErrors = metrics.NewBPFCounter(prometheus.NewDesc(
 		prometheus.BuildFQName(consts.MetricsNamespace, "", "map_errors_total"),
-		"The total number of entries dropped per LRU map.",
+		"The errors dropped per map.",
```

Comment:
```suggestion
		"The number of errors per map.",
```
This metric is just counting errors, nothing dropped.